src/status_im2/contexts/chat/messages/pin/banner/view.cljs

```diff
@@ -27,12 +27,10 @@
 
 (defn banner
   [chat-id]
-  (let [pinned-messages (rf/sub [:chats/pinned-sorted-list chat-id])
-        latest-pinned-message-id (-> pinned-messages last :message-id)
-        latest-pinned-message (get (rf/sub [:chats/chat-messages chat-id]) latest-pinned-message-id)
-        latest-pin-text (get-in latest-pinned-message [:content :parsed-text])
-        {:keys [deleted? deleted-for-me?]} latest-pinned-message
-        pins-count (count pinned-messages)
+  (let [pinned-message (rf/sub [:chats/last-pinned-message chat-id])
```

Comment:
We only need a single message, the last one, no need to subscribe to all of them